lib/core/sidenav-layout/components/header/header.component.html

```diff
@@ -0,0 +1,11 @@
+<mat-toolbar color="{{color}}">
+    <button class="mat-icon-button" mat-icon-button="" (click)="toggleMenu()">
+        <mat-icon class="mat-icon material-icons" role="img" aria-hidden="true">menu</mat-icon>
+    </button>
+
+    <img *ngIf="logo" src="{{logo}}" class="adf-app-logo">
```

Comment:
As I wrote before this was wrong in the description. We would like to have always a logo, by default the Alfresco flower, and it can be overridden by input.